spec.md

```diff
@@ -2,6 +2,15 @@
 
 The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
 
+# Abstract
+
+The OCI Runtime Specification aims to specify the inputs and outputs for running containers on various platforms.
+
+A container's input or configuration is specified as the `config.json` for the supported platforms and details fields that enable containers to run on that platform.
```

Comment:
You have “on various platforms” in the previous paragraph, so I don't think you need “for the supported platforms” here.  Maybe just echo the [current configuration description][1]?

> A container's input is a configuration which contains metadata necessary to implement standard operations against the container. This includes the process to run, environment variables to inject, sandboxing features to use, etc.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/config.md#container-configuration-file